portfolio/src/main/webapp/script.js

```diff
@@ -40,25 +40,109 @@ function addRandomGreeting() {
  * Displays the right sidebar
  */
 let isSidebarOpen = false;
-function toggleSidebar(){
+function toggleSidebar() {
     let button = document.getElementById("commentbutton");
     let el = document.getElementById("sidebar");
     (isSidebarOpen) ? closeSidebar(el, button) : openSidebar(el, button);
     isSidebarOpen = !isSidebarOpen;
 }
 
-function openSidebar(el, button){
+/**
+ * Opens the sidebar.
+ * @param {object} el - Sidebar DOM element.
+ * @param {object} button - Comment header button DOM element.
+ */
+function openSidebar(el, button) {
```

Comment:
See comments from other PR. This should be updated to be closeSidebar (same with below)